### ISSUE TITLE
site-packages/resources/roles

### ISSUE THREAD

**Issue by LumaEmu** [aacebedo/i3-xfce#3]
[Errno 2] No such file or directory: '/usr/lib/python3.5/site-packages/i3xfce/resources/roles'
I'm stuck with this when using "i3-xfce install".
All dependecies are updated(yaml, progessbar, python, etc)
Using Antergos with xfce.

**Comment by aacebedo** [comment]
Hi ! 

Try to install python3-progressbar. There is  currently an issue with this package I have to address :(

Envoyé avec Boxer

Le 12 oct. 2016 5:33 PM, LumaEmu notifications@github.com a écrit :

[Errno 2] No such file or directory: '/usr/lib/python3.5/site-packages/i3xfce/resources/roles'
I'm stuck with this when using "i3-xfce install".
All dependecies are updated(yaml, progessbar, python, etc)
Using Antergos with xfce.

**Comment by hjaf23** [comment]
Same issue here.
Any updates?

**Comment by aacebedo** [comment]
I've solved the issue on the latest release on github. However I am still working on the pypi release. Please use the archive from github install pip install

**Comment by cdhowie** [comment]
I have the same problem when installed from the latest version on Github:

```
chris@chowie-desktop:~$ sudo pip install git+https://github.com/aacebedo/i3-xfce.git
Downloading/unpacking git+https://github.com/aacebedo/i3-xfce.git
  Cloning https://github.com/aacebedo/i3-xfce.git to /tmp/pip-WCH1iK-build
  Running setup.py (path:/tmp/pip-WCH1iK-build/setup.py) egg_info for package from git+https://github.com/aacebedo/i3-xfce.git
    
Requirement already satisfied (use --upgrade to upgrade): pyyaml>=3.10 in /usr/local/lib/python2.7/dist-packages (from i3-xfce==0.6.0+1.ga0374a3)
Requirement already satisfied (use --upgrade to upgrade): progressbar2>=2.0.0 in /usr/local/lib/python2.7/dist-packages (from i3-xfce==0.6.0+1.ga0374a3)
Requirement already satisfied (use --upgrade to upgrade): ansible>=2.2.0 in /usr/local/lib/python2.7/dist-packages (from i3-xfce==0.6.0+1.ga0374a3)
Requirement already satisfied (use --upgrade to upgrade): enum>=0.4.6 in /usr/local/lib/python2.7/dist-packages (from i3-xfce==0.6.0+1.ga0374a3)
Requirement already satisfied (use --upgrade to upgrade): colorlog>=2.7.0 in /usr/local/lib/python2.7/dist-packages (from i3-xfce==0.6.0+1.ga0374a3)
Requirement already satisfied (use --upgrade to upgrade): paramiko in /usr/local/lib/python2.7/dist-packages (from ansible>=2.2.0->i3-xfce==0.6.0+1.ga0374a3)
Requirement already satisfied (use --upgrade to upgrade): jinja2 in /usr/local/lib/python2.7/dist-packages (from ansible>=2.2.0->i3-xfce==0.6.0+1.ga0374a3)
Requirement already satisfied (use --upgrade to upgrade): setuptools in /usr/local/lib/python2.7/dist-packages (from ansible>=2.2.0->i3-xfce==0.6.0+1.ga0374a3)
Requirement already satisfied (use --upgrade to upgrade): pycrypto>=2.6 in /usr/local/lib/python2.7/dist-packages (from ansible>=2.2.0->i3-xfce==0.6.0+1.ga0374a3)
Installing collected packages: i3-xfce
  Found existing installation: i3-xfce 0.6.2
    Uninstalling i3-xfce:
      Successfully uninstalled i3-xfce
  Running setup.py install for i3-xfce
    UPDATING build/lib.linux-x86_64-2.7/i3xfce/_version.py
    set build/lib.linux-x86_64-2.7/i3xfce/_version.py to '0.6.0+1.ga0374a3'
    
    Installing i3-xfce script to /usr/local/bin
  Could not find .egg-info directory in install record for i3-xfce==0.6.0+1.ga0374a3 from git+https://github.com/aacebedo/i3-xfce.git
Successfully installed i3-xfce
Cleaning up...
chris@chowie-desktop:~$ i3-xfce install
ERROR    A task failed to execute, check the messages and correct the issue before restarting i3-xfce
[Errno 2] No such file or directory: '/usr/local/lib/python2.7/dist-packages/i3xfce/resources/roles/install'
```

**Comment by aacebedo** [comment]
Hi !
Can you try this command on the git archive.
./setup.py install -prefix=<install path>

I did not have the time yet to address the issue with pip. It comes from the fact that resource files are not correctly included in the archive.

**Comment by aacebedo** [comment]
Shall be fixed with 0.6.3
Installation with pip and pypi shall work now
